examples/docker/flask/README.md

```diff
@@ -0,0 +1,31 @@
+#  App
+
+This Flask app serves a machine learning model trained on the `Walmart Recruiting - Store Sales Forecasting` dataset for predicting store sales.
+
+## Getting Started
+
+To get started with this Flask app, follow these steps:
+
+1. Run the `walmart_sales.ipynb` notebook to generate the trained model `walmart_sales_rf.joblib`.
```

Comment:
show an example using ploomber-engine to run the notebook and generate the joblib file, also include the command to install all dependencies (move them into a requirements.txt)

Reply:
[fixed](https://github.com/ploomber/doc/pull/18/commits/f42d2b13f67f982c29fe6b8d7b56e172df5b9741)